pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "todolist"
+version = "0"
+description = "devops project"
+authors = ["JoL"]
+
+[tool.poetry.dependencies]
+python = "3.8.3"
+atomicwrites = "1.4.0"
+attrs = "19.3.0"
+certifi = "2020.4.5.2"
+chardet = "3.0.4"
+click = "7.1.2"
+colorama = "0.4.3"
+Flask = "1.1.1"
+idna = "2.9"
+itsdangerous = "1.1.0"
+Jinja2 = "2.11.2"
+MarkupSafe = "1.1.1"
+more-itertools = "8.4.0"
```

Comment:
You don't need to include nested dependencies here. Just things you explicitly installed, i.e. python, flask, requests, python-dotenv 

The dev-dependencies look correct